src/Server.hs

```diff
@@ -120,7 +121,21 @@ serveGithubWebhook serveEnqueueEvent = do
       serveEnqueueEvent $ Github.CommitStatus payload
     Just "ping" ->
       serveEnqueueEvent $ Github.Ping
-    _ -> do
+    Just anEventName -> do
+      requestId <- header "X-GitHub-Hook-ID" >>= \mbRequestId ->
+        case mbRequestId of
+          Just requestId -> pure requestId
+          Nothing        -> pure "REQUEST_ID_MISSING"
+
+      -- Manually append "\n" to ensure line buffering for thread-safe logging.
+      liftIO $ Text.putStr $ LT.toStrict
+        $ "Ignored event: " <> anEventName
+        <> " - Request ID: " <> requestId <> "\n"
```

Comment:
`putStrLn` not being thread safe isn't a huge problem in practice (unless Haskell has global locks for the STDOUT streams), but this is indeed better. Nice. You can avoid the `LT.toStrict` by using the `putStr` from `Data.Text.Lazy.IO` instead.

Reply:
Thank you for the comments. Addressed both in 1f50c6f.